app/locale/ar.coffee

```diff
@@ -3089,43 +3089,43 @@ module.exports = nativeDescription: "العربية", englishDescription: "Arabi
   account:
     title: "الحساب"
     settings_title: "إعدادت الحساب"
-#    unsubscribe_title: "Unsubscribe"
-#    payments_title: "Payments"
-#    subscription_title: "Subscription"
-#    invoices_title: "Invoices"
-#    prepaids_title: "Prepaids"
-#    payments: "Payments"
-#    prepaid_codes: "Prepaid Codes"
-#    purchased: "Purchased"
-#    subscribe_for_gems: "Subscribe for gems"
-#    subscription: "Subscription"
-#    invoices: "Invoices"
+   unsubscribe_title: "إلغاء الاشتراك"
+   payments_title: "مدفوعات"
+   subscription_title: "إشتراك"
+   invoices_title: "فواتير"
+   prepaids_title: "مدفوعات المسبقة"
+   payments: "مدفوعات"
+   prepaid_codes: "رموز مدفوعة مسبقًا"
+   purchased: "تم الشراء"
+   subscribe_for_gems: "إشترك لتحصل على جواهر"
+   subscription: "إشتراك"
+   invoices: "فواتير"
 #    service_apple: "Apple"
-#    service_web: "Web"
-#    paid_on: "Paid On"
-#    service: "Service"
+   service_web: "الويب"
+   paid_on: "مدفوع في"
+   service: "خدمة"
     price: "السعر"
-#    gems: "Gems"
-#    active: "Active"
-#    subscribed: "Subscribed"
-#    unsubscribed: "Unsubscribed"
-#    active_until: "Active Until"
-#    cost: "Cost"
-#    next_payment: "Next Payment"
-#    card: "Card"
-#    status_unsubscribed_active: "You're not subscribed and won't be billed, but your account is still active for now."
-#    status_unsubscribed: "Get access to new levels, heroes, items, and bonus gems with a CodeCombat subscription!"
-#    not_yet_verified: "Not yet verified."
-#    resend_email: "Please save first then Resend email"
-#    email_sent: "Email sent! Check your inbox."
-#    verifying_email: "Verifying your email address..."
-#    successfully_verified: "You've successfully verified your email address!"
-#    verify_error: "Something went wrong when verifying your email :("
-#    unsubscribe_from_marketing: "Unsubscribe __email__ from all CodeCombat marketing emails?"
-#    unsubscribe_button: "Yes, unsubscribe"
-#    unsubscribe_failed: "Failed"
-#    unsubscribe_success: "Success"
-#    manage_billing: "Manage Billing"
+   gems: "جواهر"
```

Comment:
indentation is messed up in this PR at multiple places. You can see how `price` above has 1 more space than `gems` here. `gems` should be indented at same spacing like `price`